res/controllers/engine-api.d.ts

```diff
@@ -31,6 +31,24 @@ declare interface ScriptConnection {
     readonly isConnected: boolean;
 }
 
+/** QmlPlayerProxy */
+declare interface PlayerInfo {
+    isLoaded(): boolean;
+    track(): string;
```

Comment:
What does the string contain returened here? track() is too common to be self explaining.